src/services/dashboards/all-dashboards/modules/DashboardBoardList.vue

```diff
@@ -45,55 +55,97 @@ import {
     computed, defineComponent, reactive, toRefs,
 } from 'vue';
 
-import { PBoard, PFieldTitle, PLabel } from '@spaceone/design-system';
+import {
+    PBoard, PFieldTitle, PLabel, PPagination,
+} from '@spaceone/design-system';
+
+import { QueryHelper } from '@cloudforet/core-lib/query';
+import { SpaceConnector } from '@cloudforet/core-lib/space-connector';
 
+import { store } from '@/store';
+
+import type { DashboardItem, ScopeType } from '@/store/modules/dashboard/type';
+import { SCOPE_TYPE } from '@/store/modules/dashboard/type';
 import { FAVORITE_TYPE } from '@/store/modules/favorite/type';
 
 import FavoriteButton from '@/common/modules/favorites/favorite-button/FavoriteButton.vue';
 
 interface DashboardBoardListProps {
+    scopeType: ScopeType;
     fieldTitle: string;
     // TODO: implementation
-    dashboardList: any[];
+    dashboardList: DashboardItem[];
 }
 
 export default defineComponent<DashboardBoardListProps>({
     name: 'DashboardBoardList',
     components: {
-        PLabel, FavoriteButton, PBoard, PFieldTitle,
+        PPagination,
+        PLabel,
+        FavoriteButton,
+        PBoard,
+        PFieldTitle,
     },
     props: {
+        scopeType: {
+            type: String as PropType<ScopeType>,
+            default: undefined,
+        },
         fieldTitle: {
             type: String,
             default: undefined,
         },
         dashboardList: {
-            type: Array as PropType<any[]>,
+            type: Array as PropType<DashboardItem[]>,
             default: () => [],
         },
     },
     setup(props) {
         const state = reactive({
-            boardItemIconButtonSets: [
-                {
-                    iconName: 'ic_edit',
-                    eventAction: () => console.log('edit!'),
-                },
-                {
-                    iconName: 'ic_duplicate',
-                    eventAction: () => console.log('dup!'),
-                },
-                {
-                    iconName: 'ic_trashcan',
-                    eventAction: () => console.log('delete!'),
-                },
-            ],
-            dashboardListByBoardSets: computed(() => props.dashboardList.map((d) => ({ ...d, iconButtonSets: state.boardItemIconButtonSets }))),
+            currentPage: 1,
```

Comment:
todo: we use `thisPage` instead of `currentPage`!

Reply:
Thanks!!